react-app/src/Setting.js

```diff
@@ -27,42 +29,63 @@ export default function Setting({ db, resetTrigger }) {
     <PageTemplate>
       <NavBar setShowMenu={setShowMenu} />
       {showMenu && <SideMenu setShowMenu={setShowMenu} />}
-      <Text style={styles.title}>General</Text>
-      <View style={styles.line}>
-        <Text style={styles.text}>Remove all records</Text>
-        <Pressable style={styles.button} onPress={removeRecordsAlert}>
-          <Text style={styles.text}>Remove</Text>
-        </Pressable>
+      <View style={styles.container}>
+        <Text style={styles.title}>Setting</Text>
+        <View style={styles.buttonLayout}>
+          <Pressable style={styles.resetButton} onPress={removeRecordsAlert}>
+            <ResetIcon height={20} width={20} />
+            <Text style={styles.resetText}>Reset</Text>
+            <View style={styles.resetRightIcon}>
+              <ResetRightIcon height={20} width={20} />
+            </View>
+          </Pressable>
+        </View>
       </View>
     </PageTemplate>
   )
 }
 
 const styles = StyleSheet.create({
+  container: {
+    backgroundColor: ThemeColor.componentColor,
+    flex: 1,
+    width: null,
+    height: null,
+  },
   title: {
     color: ThemeColor.textColor,
-    fontSize: 18,
+    fontSize: 28,
     fontWeight: 'bold',
     paddingTop: 20,
-    paddingLeft: 10,
+    paddingLeft: 20,
   },
-  text: {
-    color: ThemeColor.textColor,
-    fontSize: 18,
+  buttonLayout: {
+    backgroundColor: ThemeColor.componentColor,
+    padding: 16,
+    paddingVertical: 10,
+    paddingHorizontal: 20,
   },
-  line: {
+  resetButton: {
+    backgroundColor: '#FFF',
```

Comment:
I would prefer 'backgroundColor: '#FFF''
but should I use `backgroundColor: ThemeColor.bigButtonColor'?
Then like those one:
dark mode
![IMG_7813](https://user-images.githubusercontent.com/40246308/225462483-e77088d8-4737-4a0c-92b7-e5c4a21bc1a6.PNG)
day mode
![IMG_7812](https://user-images.githubusercontent.com/40246308/225462548-a90fb802-6a7d-4974-b787-a724127c8c95.PNG)